### PR TITLE
:bar_chart: analytics tracking

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

File: app/views/search-for-your-gp.nunjucks [hunk 1]
```diff
@@ -16,7 +24,7 @@
 </ul>
 <details class="" role="group">
   <summary data-analytics="summary" role="button" aria-controls="details-content-0" aria-expanded="false">
-    <span class="details__summary">If you haven't registered for online services</span>
+    <span class="details__summary" onmousedown="dcsMultiTrack('GPAppointment', 'Not_Registered', 'WT.dl', '121')">If you haven't registered for online services</span>
```
Comment (beseven) [inline app/views/search-for-your-gp.nunjucks]: Inline JS is not something I believe is good practice, can you add a class and handle the event in JS, please? Let me know if you need any help, we've done this before on Campaigns.

Comment (ajthomascouk) [inline app/views/search-for-your-gp.nunjucks]: @beseven This came direct from Kenny. which is the way Webtrends suggest its handled.

Comment (beseven) [inline app/views/search-for-your-gp.nunjucks]: I wouldn't go with Webtrends being the forefront of best practice though. This is about the separation of the html and javascript. Anyone else have an opinion on this @neilbmclaughlin @markysoft ?

Comment (ajthomascouk) [inline app/views/search-for-your-gp.nunjucks]: I'm happy to move it out, i'm not precious to keeping it there. I'll move it out now...

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

File: app/views/search-for-your-gp.nunjucks (hunk 1 above)
Comment (neilbmclaughlin) [inline app/views/search-for-your-gp.nunjucks]: All of the analytics stuff is inline (view-source:https://beta.nhs.uk/finders/find-help). There is a rational for that in usage instructions for GA, WebTrens, etc.

I'd suggest we accept this PR and have a separate discussion about the pros and cons of changing this approach.

@beseven - are you OK with this?

Comment (beseven) [inline app/views/search-for-your-gp.nunjucks]: yeah, that sounds fine. If @ajthomascouk fixes the docker image with the envs I'm happy to look at this separately

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://gp-finder-pr-67.dev.c2s.nhschoices.net](http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-67 in nhsuk-dev succeeded (http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-67 in nhsuk-dev succeeded (http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-67 in nhsuk-dev succeeded (http://gp-finder-pr-67.dev.c2s.nhschoices.net)

**Pull request merged**